src/main/migrations/changesets/job_avoidance.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="chrisl" id="job_avoidance">
+        <comment>Adding some tracking columns for determining eligibility for Job Avoidance.</comment>
+        <addColumn tableName="EXECUTION">
+            <column name="ALLOWS_AVOIDANCE" type="BOOLEAN" defaultValueBoolean="true" >
+                <constraints nullable="false" />
+            </column>
+            <column name="DOCKER_IMAGE_HASH" type="VARCHAR(32)">
```

Comment:
There was a question about this field size. I don't think there are any hashes that would fit into 100 which won't fit into 32?